content/news/2023/09/05/data-release-als-compute-project.md

```diff
@@ -0,0 +1,23 @@
+
+In response to these challenges, the ALS Compute Project was initiated with the overarching objective of overcoming obstacles and streamlining research efforts. The project's goals include consolidating raw WGS data from all US-based ALS/FTD initiatives into a single cloud-based platform, obtaining raw WGS data for a large cohort of controls with minimal restrictions, harmonizing the resulting dataset through realignment and joint genotyping, standardizing the phenotypic dictionary of ALS/FTD case samples, and simplifying access to the harmonized cohort for the scientific community. Importantly, a major goal for ALS Compute is to increase access to ALS genetic data, especially in countries or institutions that do not have the infrastructure to handle such large datasets thus increasing the diversity of the scientific community, ultimately leading to novel discoveries.
+
+The ALS Compute Project has received strong support from the ALS/FTD WGS sequencing groups, with all agreeing to contribute their raw data. To date, over 7,000 WGS samples have been released on the AnVIL platform, and approximately 35,000 control samples have been identified for inclusion in the harmonization efforts (dbGaP:phs003184).  While interested researchers will need to apply for authorization to access these protected datasets via dbGaP, both the raw data and the metadata will be accessible through the AnVIL platform.
+
+The discovery of ALS genes has been transformative for understanding and treating this neurodegenerative disease. The ALS Compute platform, by centralizing WGS data and promoting collaboration, seeks to maintain momentum in gene discovery for ALS and FTD. The project's use of AnVIL as an open central repository for data consolidation fosters efficiency, accessibility, and inclusiveness in research efforts.
+
+By embracing the ALS Compute platform and utilizing cloud-based technology, researchers can harness the potential of harmonized data, advancing the understanding and treatment of these debilitating neurological conditions. The project sets the stage for a new era of gene discovery in ALS and FTD research, emphasizing collaboration and cost-effective approaches to benefit the scientific community and patients alike.
+
+ALS Compute was founded by John Landers, Ph.D., professor of neurology at the University of Massachusetts Chan School of Medicine; Bryan Traynor, M.D., Ph.D., senior investigator in the Laboratory of Neurogenetics at the National Institute on Aging; and Jonathan Glass, M.D., professor of neurology and pathology at Emory University School of Medicine.
+
+*This project was substantially supported by [a grant from The ALS Association](https://doi.org/10.52546/pc.gr.143413) and was recently awarded a NIH grant (R61NS13060) to further this work. More information can be found in an ALS Association [news article](https://www.als.org/stories-news/transforming-our-understanding-als-association-funded-initiative-pools-thousands).*
```

Comment:
Hi @jpaten - would you mind updating line `23` so that the link to the grant should only be around the text `grant from The ALS Association`. Thank you!